pkg/injector/healthcheck_container.go

```diff
@@ -0,0 +1,23 @@
+package injector
+
+import (
+	"os"
+
+	corev1 "k8s.io/api/core/v1"
+)
+
+func getHealthcheckContainerSpec(containerName string) corev1.Container {
```

Comment:
Nit: It looks like this is only called once where `constants.HealthcheckContainerName` is passed, so maybe it makes more sense to inline that value in this function.